### ISSUE TITLE
Could not find a version that satisfies the requirement extractcode-libarchive>=3.5.1.210122

### ISSUE THREAD

**Issue by tdruez** [nexB/scancode-toolkit#2393]
### Description

```
Collecting extractcode-libarchive>=3.5.1.210122; extra == "full" (from extractcode[full]>=21.1.21; extra == "full"->scancode-toolkit[full]==21.2.9)
  Could not find a version that satisfies the requirement extractcode-libarchive>=3.5.1.210122; extra == "full" (from extractcode[full]>=21.1.21; extra == "full"->scancode-toolkit[full]==21.2.9) (from versions: 3.1.4, 3.1.5)
No matching distribution found for extractcode-libarchive>=3.5.1.210122; extra == "full" (from extractcode[full]>=21.1.21; extra == "full"->scancode-toolkit[full]==21.2.9)
```

### How To Reproduce

`pip install scancode-toolkit[full]==21.2.9`

### System configuration

On a Ubuntu 20.04 container:

```
jobs:
  build:
    runs-on: ubuntu-20.04
```

**Comment by tiulpin** [comment]
Strange, had almost the same problem on macOS yesterday https://github.com/nexB/scancode-plugins/issues/4

Previous versions worked, with the new version it's not possible to find the wheels anymore

**Comment by tdruez** [comment]
@pombredanne after further testing, the issue only occurs with Python3.6.
Everything installs and works fine with 3.7, 3.8, and 3.9

**Comment by tdruez** [comment]
It's actually a pip issue using the old Python3.6 pip version, upgrading pip to latest version first solves it:
`pip install --upgrade pip`

**Comment by pombredanne** [comment]
@tdruez I am relieved... as I was not able to reproduce. ScanCode is now using pip 20.3+ or 21.x
@tiulpin can you check if this is your issue too?

**Comment by tiulpin** [comment]
I checked the issue on Ubuntu Docker, was not able to reproduce it now

**Comment by pombredanne** [comment]
This should be all Fixed in v21.2.25
